website/static/js/comment.js

```diff
@@ -119,6 +143,23 @@ BaseComment.prototype.setupToolTips = function(elm) {
     });
 };
 
+BaseComment.prototype.get_url = function() {
```

Comment:
It looks like `get_url` isn't being used anywhere. Did you mean to use it in `BaseComment.prototype.fetch`?